lib/connection/bind_uploader.js

```diff
@@ -69,27 +71,14 @@ function BindUploader(options, services, connectionConfig, requestId)
 			}
 			strbuffer += '\n';
 
-			if (curBytes < MAX_BUFFER_SIZE)
+			if ((strbuffer.length >= MAX_BUFFER_SIZE) || (i == bindings.length -1))
```

Comment:
Because of this we don't create multiple files anymore...  Before this we would have 10 files in our test case by default, now we have one...

Is this intentional?

Reply:
we have bug here before, the file size is too small, even we set it to 100m